maintainers/scripts/hydra-eval-failures.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -i python -p pythonFull pythonPackages.requests pythonPackages.pyquery pythonPackages.click
+#!nix-shell -i python3 -p python3Full python3Packages.requests python3Packages.pyquery python3Packages.click
```

Comment:
`python3.withPackages(ps: with ps; [ requests pyquery click ])`

Reply:
???

The closest I've been able to get to what you suggest is 

    nix-shell -E 'with import <nixpkgs> {}; [(python3.withPackages(ps: with ps; [ requests pyquery click ])) python3Full]'

but that fails with "a single derivation is required"

it seems that it should be possible to do 

    nix-shell -p 'python35.withPackages(ps: with ps; [ numpy toolz ])'

but [the documentation](https://github.com/NixOS/nixpkgs/blame/9d9a28f/doc/languages-frameworks/python.md#L115) is wrong, since I get this error:

    undefined variable ‘python35’ at (string):1:1

(same with python36)

Reply:
@FRidh done